src/sql/parts/notebook/notebook.component.ts

```diff
@@ -153,6 +155,19 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		}
 	}
 
+	public unselectActiveCell() {
+		if (!this._cellSelected) {
+			if (this._activeCell) {
+				this._activeCell.active = false;
+			}
+			this._activeCell = null;
```

Comment:
Minor: can we have a `clearActiveCell` method for this? We might use it outside click handler in the future.